source/common/secret/secret_manager_impl.h

```diff
@@ -35,9 +36,18 @@ class SecretManagerImpl : public SecretManager, Logger::Loggable<Logger::Id::sec
       const envoy::api::v2::core::ConfigSource& config_source, const std::string& config_name,
       Server::Configuration::TransportSocketFactoryContext& secret_provider_context) override;
 
+  CertificateValidationContextConfigProviderSharedPtr
+  findOrCreateCertificateValidationContextProvider(
+      const envoy::api::v2::core::ConfigSource& config_source, const std::string& config_name,
+      Server::Configuration::TransportSocketFactoryContext& secret_provider_context) override;
+
 private:
-  // Remove dynamic secret provider which has been deleted.
+  // Removes dynamic secret provider which has been deleted.
   void removeDynamicSecretProvider(const std::string& map_key);
+  // Finds or creates SdsApi object.
+  SdsApiSharedPtr innerFindOrCreate(
```

Comment:
Can we rename `innerFindOrCreate` to something else? Even `findOrCreate` would be fine, I just find "inner" leads to more questions (e.g. "is this an inner class?", "inner of what?"), so this isn't as clear as it could be.

Reply:
Good point. Changed to findOrCreate. Thanks.